.github/workflows/ci.yml

```diff
@@ -505,6 +508,12 @@ jobs:
       upgrade-to-newer-dependencies: ${{ needs.build-info.outputs.upgrade-to-newer-dependencies }}
       include-success-outputs: ${{ needs.build-info.outputs.include-success-outputs }}
       debug-resources: ${{ needs.build-info.outputs.debug-resources }}
+      special-tests-required: ${{
```

Comment:
As discussed below - we can likely remove it back completely  - it's a duplication with the above if now, and I bet we will forget to update it in the future if anything changes, so better to remove it.

Reply:
sorry my bad in misinterpreted :)